ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java

```diff
@@ -36,10 +36,13 @@ public enum SqlExample {
       "alter table db_name.table_name add column data int ;\n"
           + "alter table db_name.table_name alter column data bigint ;\n"
           + "alter table db_name.table_name drop column data;"),
-  ALTER_PROPERTIES(
-      "AlterProperties",
+  ADD_PROPERTIES(
+      "AddProperties",
       "alter table db_name.table_name set tblproperties (\n"
           + "    'comment' = 'A table comment.');"),
+  DROP_PROPERTIES(
+      "DropProperties",
```

Comment:
If we use `SetProperties`, maybe we should use `UnsetProperties` here.

Reply:
Plz review it again.